config/manifest-study.rb

```diff
@@ -1,5 +1 @@
-render "docker-compose-study.yaml.ctmpl"
 render "pepperConfig.js.ctmpl"
-copy_file "study-nginx.conf"
-copy_secret_from_path "secret/pepper/#{$env}/#{$version}/#{$study_key}/cert.key"
-copy_secret_from_path "secret/pepper/#{$env}/#{$version}/#{$study_key}/cert.crt"
```

Comment:
@zyme We can say bye to this stuff now, yes?

Reply:
Yes. 👋